spec/headers_spec.rb

```diff
@@ -7,14 +7,6 @@
       subject(:headers) { described_class.new(request) }
       let(:uri) { '' }
 
-      context 'empty uri' do
```

Comment:
I this is a probably important spec that describes an edge case

Reply:
I don't think that this case is essentiel because the next case is pretty the same thing. However, I agree to add a test for that if you found me a Gem that allow this. Net::HTTP rand RestClient v2 raise an exeception if you use a empty string.

Reply:
You're right. This test was [always specific to RestClient](https://github.com/mgomes/api_auth/commit/6c957342f830a3f1ccc3a326bd87220fbe59e4a2#diff-e8b2590d477903fed0685b0f801c057dL307), and since RestClient no longer allows it, the test provides no value and only causes failure.